README.DropDownFilter.md

```diff
@@ -0,0 +1,40 @@
+# Drop down filter
+Enable users to filter a list view at run time, with various options that can be selected from a drop-down
+
+## Features
+* Filter items by an attribute
+* Filter items by XPath
+* Select a filter from a list of options
+* Set a default filter option
+
+## Dependencies
+Mendix 7.6
+
+## Demo project
```

Comment:
Not `Demo project` => `Sample`.
Section demo project is for the URL to the sandbox
Please update for versions

Reply:
Done